swap_meet/item.py

```diff
@@ -1,2 +1,27 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self,category=None,condition=0,age=0):
+        self.condition=condition
+        self.age=age
+        if category == None:
+            self.category = ""
+        else:
+            self.category = category    
```

Comment:
Excellent work! None is often used as a default argument value in Python because it allows us to call the function without providing a value for an argument that isn't required on each function invocation. Here's the article I referenced: https://bobbyhadz.com/blog/python-using-none-as-default-argument#:~:text=None%20is%20often%20used%20as,for%20list%20and%20dict%20arguments.

Reply:
You could refactor this to a shorthand (one-liner): `self.category = category if category is not None else `